cfgov/unprocessed/css/organisms/email-popup.less

```diff
@@ -66,7 +66,7 @@
 
     .respond-to-max( @bp-xs-max, {
       background: none;
-      .m-form-field-with-button .btn {
+      .m-form-field-with-button .a-btn {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/e8b89a9d7072510b74c7fe980cebe22c7a892cca